srcpkgs/sway/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'sway'
 pkgname=sway
 version=1.7
-revision=1
+revision=2
 build_style=meson
 configure_args="-Dwerror=false -Db_ndebug=false"
 conf_files="/etc/sway/config"
 hostmakedepends="pkg-config wayland-devel scdoc git"
```

Comment:
why is git required here?

Reply:
Doesn't seem like it's required; build config uses it to get hash and branch name

Reply:
in that case, please drop git from hostmakedepends, this is built from a tarball, not a git repository, so it won't work anyway

Reply:
done